recipes/pds.api-client/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pds.api-client" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pds_api_client-{{ version }}.tar.gz
+  sha256: 24c2f007d14069a082143a3d55a94dfe03049f59dc38013a9d888370fb06e462
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - poetry-core
+    - pip
+  run:
+    - python >=3.7.0,<4.0.0
+    - urllib3 >=1.25.3,<=2.2.3
```

Comment:
For the package that's up on pypi it has:

```
REQUIRES = [
    "urllib3 >= 1.25.3, < 2.1.0",
    "python-dateutil",
    "pydantic >= 2",
    "typing-extensions >= 4.7.1",
]
```

so you're getting the wrong version of `urllib3` that goes outside of that bound in the `pip check`. Strangely there are conflicting versions in `pyproject.toml`:

```
[tool.poetry.dependencies]
python = "^3.7"

urllib3 = ">= 1.25.3"
python-dateutil = ">=2.8.2"
pydantic = ">=2"
typing-extensions = ">=4.7.1"
```